luigi_pipeline/seqr_mito_loading.py

```diff
@@ -0,0 +1,45 @@
+import logging
+import sys
+
+import luigi
+import hail as hl
+
+from lib.model.mito_mt_schema import SeqrMitoVariantsAndGenotypesSchema, SeqrMitoVariantSchema, SeqrMitoGenotypesSchema
+from luigi_pipeline.seqr_loading_optimized import SeqrVCFToVariantMTTask, BaseVCFToGenotypesMTTask, BaseMTToESOptimizedTask
+
+logger = logging.getLogger(__name__)
+
+
+class SeqrMitoVariantMTTask(SeqrVCFToVariantMTTask):
+    """
+    Loads all annotations for the variants of a Matrix Table into a Hail Table.
+    """
+    high_constraint_interval_path = luigi.Parameter(description='Path to the tsv file storing the high constraint intervals.')
+    RUN_VEP = False
+    SCHEMA_CLASS = SeqrMitoVariantSchema
+
+    def get_schema_class_kwargs(self):
+        kwargs = super().get_schema_class_kwargs()
+        kwargs['high_constraint_region'] = hl.import_locus_intervals(self.high_constraint_interval_path,
+                                                                     reference_genome='GRCh38')
+        return kwargs
+
+    def import_dataset(self):
+        mt = hl.read_matrix_table(self.source_paths[0])
+        return mt.select_globals('vep_version')
```

Comment:
what does this do?

Reply:
I thought we might need the `vep_version` global field. I will remove it after changing `annotate_globals` to `select_globals`.